use/wordcount-example.md

```diff
@@ -0,0 +1,334 @@
+* Reading input (in this example: reading text files)
+* Applying ParDo transforms
+* Applying SDK-provided transforms (in this example: Count)
+* Writing output (in this example: writing to a text file)
+* Running the Pipeline
+
+The following sections explain these concepts in detail along with excerpts of the relevant code from the Minimal WordCount pipeline.
+
+### Creating the Pipeline
+
+The first step in creating a Beam pipeline is to create a Pipeline Options object. This object lets us set various options for our pipeline, such as the pipeline runner that will execute our pipeline, the ID of our project, and the staging location for the pipeline to store its files (used to make your jars accessible in the Cloud). In this example we set these options programmatically, but more often command-line arguments are used to set Pipeline options.
+
+In our example, we selected `DataflowRunner` for our Beam runner. This means our pipeline will be executed in the cloud using Google Cloud's Dataflow Service. There are other runners you can specify for executing your pipeline, such as `FlinkRunner` or `SparkRunner`. If you omit specifying a runner, your pipeline will be executed locally.
+
+```
+ PipelineOptions options = PipelineOptionsFactory.create();
+
+    // In order to run your pipeline, you need to make following runner specific changes:
+    //
+    // CHANGE 1/3: Select a Beam runner, such as DataflowRunner or FlinkRunner.
+    // CHANGE 2/3: Specify runner-required options.
+    // For DataflowRunner, set project and temp location as follows:
+    //   DataflowPipelineOptions dataflowOptions = options.as(DataflowPipelineOptions.class);
+    //   dataflowOptions.setRunner(DataflowRunner.class);
+    //   dataflowOptions.setProject("SET_YOUR_PROJECT_ID_HERE");
+    //   dataflowOptions.setTempLocation("gs://SET_YOUR_BUCKET_NAME_HERE/AND_TEMP_DIRECTORY");
+    // For FlinkRunner, set the runner as follows. See {@code FlinkPipelineOptions}
+    // for more details.
+    //   options.setRunner(FlinkRunner.class);
+```
+
+The next step is to create a Pipeline object with the options we've just constructed. The Pipeline object builds up the graph of transformations to be executed, associated with that particular pipeline.
+
+```
+Pipeline p = Pipeline.create(options);
+```
+
+### Applying Pipeline Transforms
+
+The Minimal WordCount pipeline contains several transforms to read data into the pipeline, manipulate or otherwise transform the data, and write out the results. Each transform represents an operation in the pipeline.
+
+Each transform takes some kind of input (data or otherwise), and produces some output data. The input and output data is represented by the SDK class `PCollection`. `PCollection` is a special class, provided by the Beam SDK, that you can use to represent a data set of virtually any size, including infinite data sets.
+
+![alt tag](https://cloud.google.com/dataflow/images/pipeline-dataflow2.png)
```

Comment:
Should we upload separate copies of these pictures instead of linking to these?

@jbonofre Any issues with us using the same images here as we do on cloud.google.com? Conceptually this is the documentation that went along with the code we donated.